test/lib/jdk/test/lib/security/artifacts/ThirdPartyArtifacts.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package jdk.test.lib.security.artifacts;
+
+import jdk.test.lib.artifacts.Artifact;
+import jdk.test.lib.artifacts.ArtifactResolver;
+import jdk.test.lib.artifacts.ArtifactResolverException;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class ThirdPartyArtifacts {
+
+    public static final String ACVP_BUNDLE_LOC = "jpg.tests.jdk";
+    public static final String ACVP_BUNDLE_NAME = "ACVP-Server";
+    public static final String ACVP_BUNDLE_VERSION = "1.1.0.38";
+
+    // Version of the NSS artifact. This coincides with the version of
+    // the NSS version
+    public static final String NSS_BUNDLE_VERSION = "3.107";
+    public static final String NSSLIB = "jpg.tests.jdk.nsslib";
+
+    @Artifact(
+            organization = NSSLIB,
+            name = "nsslib-windows_x64",
+            revision = NSS_BUNDLE_VERSION,
+            extension = "zip")
+    public static class WINDOWS_X64 {
```

Comment:
Please rename the NSS static classes to `NSS_<platform>` as ThirdPartyArtifacts will also contain other artifacts for these platforms
```suggestion
    public static class NSS_WINDOWS_X64 {
```

Reply:
done in the next commit